peer.go

```diff
@@ -276,6 +276,33 @@ func (p *peer) Start() error {
 	return nil
 }
 
+// getEdgeInfo returns the channel edge info.
+// TODO(andrew.shvv) place it inside the router after it will be clear that
+// this method is needed inside the channel link.
+func (p *peer) getEdgeInfo(chanID lnwire.ChannelID) (*channeldb.ChannelEdgePolicy,
```

Comment:
Why does this method need to reside as part of the peer struct? It seems that instead a closure should be created at the site of the initialization of the configuration for a channel link.